src/commands/variables.rs

```diff
@@ -1,21 +1,40 @@
 use super::*;
 use crate::{
-    controllers::project::{ensure_project_and_environment_exist, get_project},
-    controllers::variables::get_service_variables,
+    consts::TICK_STRING,
+    controllers::{
+        environment::get_matched_environment,
+        project::{ensure_project_and_environment_exist, get_project},
+        variables::get_service_variables,
+    },
     errors::RailwayError,
     table::Table,
 };
+use anyhow::bail;
+use std::{collections::BTreeMap, time::Duration};
 
 /// Show variables for active environment
 #[derive(Parser)]
 pub struct Args {
-    /// Service to show variables for
+    /// The service to show/set variables for
     #[clap(short, long)]
     service: Option<String>,
 
+    /// The environment to show/set variables for
+    #[clap(short, long)]
+    environment: Option<String>,
+
     /// Show variables in KV format
     #[clap(short, long)]
     kv: bool,
+
+    /// The "{key}={value}" environment variable pair to set the service variables.
+    /// Example:
+    ///
+    /// ```bash
```

Comment:
The quotes shouldn't be here as it doesn't render properly with `--help`

![image](https://github.com/user-attachments/assets/c3be4735-661c-43a0-a673-7905f0d93bb5)